pyproject.toml

```diff
@@ -14,7 +14,7 @@ homepage = "https://hathor.network/"
 repository = "https://github.com/HathorNetwork/tx-mining-service/"
 
 [tool.poetry.dependencies]
-python = ">=3.6,<4.0"
+python = ">=3.7,<4.0"
```

Comment:
Since we only build for python 3.9 (Fixed on the Dockerfile), why not make this `>=3.9,<4.0`?

Reply:
I'm ok with this, but this would require people to have Python 3.9 in their local environments to run the project.

I'm not sure this would not pose a challenge to someone.

What do you think?

Reply:
If the PR validation builds and tests on 3.9 I don't think it's an issue.